### DISCUSSION TITLE
Circuit overseer visit week (Lack of translation)

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#2314] [Ideas] [answered]
### Is your idea related to a problem?

_Just to complement, this problem, with the names of the meeting sections, occurs in all mid-week meetings. And not just during the week of the visit as mentioned._

### Describe your idea

At the midweek meeting, when I indicate that that week is the circuit overseer's visit, the midweek meeting titles are not translated. (Lack of translation into PT-BR)

### Describe alternatives you've considered

In the current version, everything is fine, just when you go to the settings and add the date of the SC visit

### Additional context

![image](https://github.com/sircharlo/meeting-media-manager/assets/90984283/5d22d675-a584-45c9-a3de-fba5e59aeec7)

**Comment by mtdvlpr** [comment, marked as answer]
@e-Feitosa the problem is that the headings can't be extracted from the new workbook format (yet). You could add the headings manually be editing the `AppData/Roaming/meeting-media-manager/Publications/T/mwb/headings.json` file

**Reply by e-Feitosa** [reply]
I just didn't find any files (headings.json) inside the mentioned folder (mwb).
⬇️
\AppData\Roaming\meeting-media-manager\Publications\T
![image](https://github.com/sircharlo/meeting-media-manager/assets/90984283/04a44a9b-2e61-4252-b3af-4192b9af5481)

**Reply by mtdvlpr** [reply]
@e-Feitosa in that case you can create the file and add the following in it:

```js
{
  "treasures": "translated treasures title",
  "apply": "translated apply title",
  "living": "translated living title",
}
```
